### PR TITLE
Add nested form resources.

### PR DESCRIPTION
**Pull request by domita**

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -27,6 +25,8 @@ def new
       category_elements.push(category.id)
       @category_options.push(category_elements)
     end
+
+     1.times {@course.resources.build}
```
Comment (ndebiec) [inline app/controllers/courses_controller.rb]: @course.resources.build

Comment (ndebiec) [reply app/controllers/courses_controller.rb]: *why not?

**Review by ndebiec — APPROVED** [review]
Amazing job!

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (domita) [reply app/controllers/courses_controller.rb]: sounds good :)

**Pull request merged**